libs/qwikdev-astro/src/index.ts

```diff
@@ -115,7 +121,16 @@ function hash() {
 async function moveArtifacts(srcDir: string, destDir: string) {
   await mkdir(destDir, { recursive: true });
   for (const file of await readdir(srcDir)) {
-    await rename(join(srcDir, file), join(destDir, file));
+    let destPath = join(destDir, file);
+    if (existsSync(destPath)) {
+      if (statSync(destPath).isDirectory()) {
+        destPath = getUniqueTempDir(destPath);
```

Comment:
I am confused why you have `getUniqueTempDir` here. There should only be only one place where the temp directory is being created.